pkg/k8s/commands/run.go

```diff
@@ -47,8 +47,14 @@ func Run(ctx context.Context, args []string, opts flag.Options) error {
 	case clusterArtifact:
 		return clusterRun(ctx, opts, cluster)
 	case allArtifact:
+		if opts.Format == types.FormatCycloneDX {
+			return xerrors.Errorf("Error: KBOM with CycloneDX format is not supported for all namespace scans")
```

Comment:
It's obviously an error. I don't think we need the `Error:` prefix.

```suggestion
			return xerrors.Errorf("KBOM with CycloneDX format is not supported for all namespace scans")
```